src/ripple/ripple.js

```diff
@@ -118,7 +119,12 @@ MaterialRipple.prototype.upHandler_ = function(event) {
 
   // Don't fire for the artificial "mouseup" generated by a double-click.
   if (event && event.detail !== 2) {
-    this.rippleElement_.classList.remove(this.CssClasses_.IS_VISIBLE);
+    // Allow a repaint to occur before removing this class, so the animation
+    // shows for tap events, which seem to trigger a mouseup too soon after
+    // mousedown.
+    window.setTimeout(function() {
```

Comment:
I was under the impression that a timeout of 0 was sufficient as well, but for the sake of consistency maybe just merge this PR, and then drop all instances of `TINY_TIMEOUT` in another commit, and abstract out all the individual `setTimeout` calls to a shared helper method.

I think you can actually do `setTimeout(fn)`, i.e. ommit the second argument, and it defaults to 0.